src/js-marketing-notification.ts

```diff
@@ -1,32 +1,31 @@
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
 export interface Notification {
-  tabContent: string,
-  heading: string;
-  buttonTitle: string;
-  image?: string;
-  content?: string;
+  tabContent: Element,
+  notificationBlocks: Array<Element>
 }
 
 export class JsMarketingNotification {
   private notificationContainer: Element;
-  
-  constructor(private notificationTitle: string) {}
+  private lsm = new LocalStorageManager();
 
-  public output(notification: Notification): void {
-    const lsm = new LocalStorageManager(),
-          body = this.createNotificationBody(notification);
+  constructor(private notificationTitle: string) { }
 
+  public output(notification: Notification): void {
     this.notificationContainer = document.createElement('div');
     this.notificationContainer.classList.add('marketing-notification');
+
+    const tab = this.createNotificationTab(notification.tabContent, this.notificationContainer),
+          body = this.createNotificationBody(notification.notificationBlocks);
+
+    this.notificationContainer.appendChild(tab);
```

Comment:
Since you are only using `tab` and `body` once, you should just do it all inline, no need for a variable:

`this.notificationContainer.appendChild(this.createNotificationTab(notification.tabContent, this.notificationContainer), 
);`